src/Service/DynamoDb/src/Result/TableExistsWaiter.php

```diff
@@ -11,18 +11,14 @@
 class TableExistsWaiter extends Waiter
 {
     protected const WAIT_TIMEOUT = 500.0;
-    protected const WAIT_DELAY = 20.0;
+    protected const WAIT_DELAY = 5.0;
 
     protected function extractState(Response $response, ?HttpException $exception): string
     {
         if (200 === $response->getStatusCode() && 'ACTIVE' === ($response->toArray()['Table']['TableStatus'] ?? null)) {
             return self::STATE_SUCCESS;
         }
 
-        if (null !== $exception) {
```

Comment:
This condition is the same as the following ternary condition.

Reply:
please open an issue.

when the last check is about the exception, the ternary is not needed